src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1990,6 +1990,44 @@ limitations under the License.
             <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
         </clusters>
     </deviceType>
+
+    <deviceType>
+        <name>MA-basic-videoplayer</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Basic Video Player</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0028</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+                <requireCommand>IdentifyQuery</requireCommand>
+                <requireCommand>TriggerEffect</requireCommand>
+            </include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
+            </include>
+            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
```

Comment:
Why is this including the binding client?

Reply:
Addressed in: https://github.com/project-chip/connectedhomeip/pull/21045